test/e2e/kuttl/ScaleUpAndDownDC/01-scaleUp-dc.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: db.orange.com/v1alpha1
+kind: CassandraCluster
+metadata:
+  name: cassandra-e2e
+spec:
+  nodesPerRacks: 1
+  topology:
```

Comment:
you can just update the one at line 6 and remove the topology here

Reply:
I think we can keep it this way but add more steps like : setup a 2 dc topo, scale 1 up at dc level et assert only this one is correctly scaleup, then scaleup at root level which should scaleup only the second dc and then scale down both with assertions. What do you think if I create such a flow ?

Reply:
Ok lets focus on basic scaleup for this PR for now. Changes applied !